createdb

```diff
@@ -0,0 +1,40 @@
+#!/bin/sh
+
+set -e -u
+
+./services
+
+mysql -u root -ppassword <<\____
+
+    # Create the metasmoke database and metasmoke_test database
+
+    CREATE DATABASE metasmoke;
```

Comment:
This is what `rails db:create` does, you can drop it from the raw SQL.

Reply:
Be clarify what are you saying, that does not make sense :-)

Reply:
This was just a copy/paste from the instructions in the wiki or the README (don't remember exactly now) but thanks, I'll try to fix it.

Reply:
There's some sort of dependency here, I can't remember exactly how I arrived at this workaround, but if I remove the SQL, I get permission denied from `db:create`.

```
Starting MariaDB database server: mysqld.
Starting redis-server: redis-server.
/usr/src/app/config/initializers/appconfig.rb:3: warning: already initialized constant AppConfig
/usr/src/app/config/environments/development.rb:21: warning: previous definition of AppConfig was here
Access denied for user 'metasmoke'@'localhost'
Couldn't create database for {"adapter"=>"mysql2", "database"=>"metasmoke", "encoding"=>"utf8", "username"=>"metasmoke", "password"=>"password", "host"=>"127.0.0.1", "port"=>3306}
rails aborted!
Mysql2::Error: Access denied for user 'metasmoke'@'localhost'
/usr/local/bundle/gems/mysql2-0.4.10/lib/mysql2/client.rb:89:in `connect'
/usr/local/bundle/gems/mysql2-0.4.10/lib/mysql2/client.rb:89:in `initialize'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/mysql2_adapter.rb:22:in `new'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/mysql2_adapter.rb:22:in `mysql2_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:809:in `new_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:853:in `checkout_new_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:832:in `try_to_checkout_new_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:793:in `acquire_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:521:in `checkout'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:380:in `connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_adapters/abstract/connection_pool.rb:1008:in `retrieve_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_handling.rb:118:in `retrieve_connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/connection_handling.rb:90:in `connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/mysql_database_tasks.rb:6:in `connection'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/mysql_database_tasks.rb:14:in `create'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/database_tasks.rb:119:in `create'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/database_tasks.rb:139:in `block in create_current'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/database_tasks.rb:316:in `block in each_current_configuration'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/database_tasks.rb:313:in `each'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/database_tasks.rb:313:in `each_current_configuration'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/tasks/database_tasks.rb:138:in `create_current'
/usr/local/bundle/gems/activerecord-5.2.0.rc1/lib/active_record/railties/databases.rake:29:in `block (2 levels) in <top (required)>'
/usr/local/bundle/gems/railties-5.2.0.rc1/lib/rails/commands/rake/rake_command.rb:23:in `block in perform'
/usr/local/bundle/gems/railties-5.2.0.rc1/lib/rails/commands/rake/rake_command.rb:20:in `perform'
/usr/local/bundle/gems/railties-5.2.0.rc1/lib/rails/command.rb:48:in `invoke'
/usr/local/bundle/gems/railties-5.2.0.rc1/lib/rails/commands.rb:18:in `<top (required)>'
bin/rails:11:in `require'
bin/rails:11:in `<main>'
Tasks: TOP => db:create
(See full trace by running task with --trace)
The command '/bin/sh -c gem install bundler     && bundle install     && ./createdb     && sed -i 's/^web:.*/& -b 0.0.0.0/' Procfile     && yarn install' returned a non-zero code: 1
```